src/Components/test/testassets/BasicTestApp/wwwroot/js/jsinteroptests.js

```diff
@@ -219,6 +235,42 @@ function returnArrayAsync() {
   });
 }
 
+function roundTripByteArray(byteArray) {
+  if (byteArray.constructor !== Uint8Array) {
```

Comment:
I believe you can use `instanceof` here instead of checking the constructor that way.

Reply:
I had tried going that route, but it didn't appear to be sufficient as it just defaulted to `object` instead of `Uint8Array`
![image](https://user-images.githubusercontent.com/14852843/120687706-1cd74180-c457-11eb-8d66-48a7f3debb8e.png)